e2e/ldap.openldap.json

```diff
@@ -0,0 +1,4 @@
+{
+    "ldapServer": "ldap.e2e.dev.spinmint.com",
+    "ldapPort": 389
+}
```

Comment:
I suggest not to create this file. We'll handle this in CI by setting up environment variable as you mentioned in https://github.com/mattermost/mattermost-webapp/pull/5114/files#diff-2310db0264dfafc826f0b694330fe582R15-R16